monitoring/deployment/postgresql-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
```

Comment:
This should be under the folder structure,
/monitoring/grafana-postgres/statefulset.yaml

Reply:
made changes